lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -99,7 +104,38 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
   if (isset($vars['content']['zendesk_form'])) {
     $vars['zendesk_form'] = $vars['content']['zendesk_form'];
   }
+}
+
+/**
+ * Preprocess variables for hot module.
+ */
+function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
+  $node = node_load($vars['nid']);
+  $goal = dosomething_helpers_get_variable('node', $vars['nid'], 'goal');
+  $rb_progress = dosomething_reportback_get_reportback_total_by_nid($node->nid);
+  $rb_percent = ($rb_progress / $goal) * 100;
+
+  if($rb_percent <= 20) {
```

Comment:
should we assign a default value of 0 instead so there is always a progress copy?